src/ORM/Table.php

```diff
@@ -243,6 +247,16 @@ public function __construct(array $config = [])
         if (!empty($config['associations'])) {
             $associations = $config['associations'];
         }
+        if (!empty($config['validator'])) {
+            if (!is_array($config['validator'])) {
+                $this->validator(null, $config['validator']);
+            } else {
+                foreach ($config['validator'] as $key => $validator) {
+                    $name = (!is_int($key)) ? $key : null;
```

Comment:
`null` is not a value value of  `$name` param. There's no need to support numeric index, one can just use `'default'` key.